### PR TITLE
Some general improvements

### PR DESCRIPTION
**Pull request by BeastyBlacksmith**
Maybe this is a bit early in the process, but these are some things that I found irritating on my first read and what I would write instead.

### CODE REVIEW

File: competencies.md [hunk 1]
```diff
@@ -119,10 +120,10 @@ established research culture and the commitment to being part of a team.
 
 There are a number of common terms that are used widely within this paper but we
 recognise that they may be understood differently depending on the national research
-environments and processes that readers are familiar with. So, a few defintions:
+environments and processes that readers are familiar with. So, a few definitions:
```
Comment (BeastyBlacksmith) [inline competencies.md:123]: I think that calling two "a number of terms" is a bit of a stretch but probably there might come more?

Also I think there should be a clearer separation from the main text, maybe little paragraph headings or something like `**software**: `

File: competencies.md [hunk 2]
```diff
@@ -144,32 +145,32 @@ on the development of research software.
...
 provide the guiding principles for the work of an RSE.
-Section 3 condenses and generalizes 
+Section 3 condenses and generalizes
 the workshop discussions into a set of generic skills.
-The skills themselves fall into three categories 
+The skills themselves fall into three categories
 'software engineering', 'research' and 'team skills'
 reflecting the hybrid nature of an RSE.
 To exemplify the generic skills,
-we also list some current day tasks
```
Comment (jpthiele) [inline competencies.md:165]: "current day" as in "as it is right now" was the correct wording,
but should be reworded if you stumbled over it

File: competencies.md [hunk 3]
```diff
@@ -181,25 +182,25 @@ like bioinfomatics.
...
+national facilities is becoming readily available. Additionally, pressure is
+growing from funding bodies to prioritise projects that generate archived,
+annotated, re-usable and potentially remotely executable data. These resources
 and requirements fall within the skill set of RSEs. They become a vital link to
 cross-pollinate computational skills and infrastructure know-how between domain
 scientists. Funders and research managers will find the discussion in this paper
 valuable in order to observe how software development in academia will be
 institutionalised. Finally, the strong emphasis on team skills allows RSEs to be
-very employable in workplaces outside academia.
+well prepared to common challanges in workplaces outside academia.
```
Comment (jpthiele) [inline competencies.md:203]: typo: challenge

File: competencies.md (hunk 2 above)
Comment (BeastyBlacksmith) [reply competencies.md:165]: It sounded like things that you do at daytime to me

Comment (BeastyBlacksmith) [reply competencies.md:165]: just "current" should be fine then

Comment (jpthiele) [reply competencies.md:165]: True. I think that's good!

File: competencies.md (hunk 1 above)
Comment (jpthiele) [reply competencies.md:123]: maybe just leave out 'a number of' to make it 'There are common terms'?

**Pull request merged**